README.md

```diff
@@ -82,25 +100,41 @@ from [Frontegg Portal Domain](https://portal.frontegg.com/development/settings/d
 ```groovy
     dependencies {
     // Add the Frontegg Android Kotlin SDK
-    implementation 'com.frontegg.sdk:android:1.+'
+    implementation 'com.frontegg.sdk:android:LATEST_VERSION'
+    // Add Frontegg observables dependency
+    implementation 'io.reactivex.rxjava3:rxkotlin:3.0.1'
+}
+```
+```kotlin
+    dependencies {
+    // Add the Frontegg Android Kotlin SDK
+    implementation ("com.frontegg.sdk:android:LATEST_VERSION")
     // Add Frontegg observables dependency
     implementation 'io.reactivex.rxjava3:rxkotlin:3.0.1'
 }
 ```
 
 ### Set minimum sdk version
 
-To set up your Android minimum sdk version, open root gradle file at`android/build.gradle`,
+To set up your Android minimum sdk version, open the root gradle file at`android/build.gradle`,
 and add/edit the `minSdkVersion` under `buildscript.ext`:
 
 ```groovy
 buildscript {
     ext {
-        minSdkVersion = 26
+        minSdk = 26
         // ...
     }
 }
 ```
+```kotlin
+android {
```

Comment:
https://github.com/frontegg/frontegg-android-kotlin/pull/78/files#r1790822624 is why I added an `android` block. `buildscript` is still in the Groovy block above.